openfl/component/aggregator/aggregator.py

```diff
@@ -1005,6 +1067,7 @@ def _end_of_round_check(self):
         # Cleaning tensor db
         self.tensor_db.clean_up(self.db_store_rounds)
         # Reset straggler handling policy for the next round.
+        #E.L should it be saved?
         self.straggler_handling_policy.reset_policy_for_round()
```

Comment:
Same as above, I'd suggest. Otherwise we could run into weird corner cases in straggler scenarios combined with aggregator recovery.